index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend:wght@400;500&family=Manrope&family=Noto+Color+Emoji&display=swap" rel="stylesheet">
+    <link rel = "stylesheet" href="index.css">
+    <script type="module" src = "index.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@1.27.0/build/global/luxon.min.js"></script>
+    <title>Awesome Books</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h3><a href = "index.html">Awesome Books</a></h3>
+          <ul>
+            <li><a id="book-link" href="#">List</a></li>
+            <li class = "line"><a id="add-link" href="#">Add Book</a></li>
+            <li class = "line"><a id="contact-link" href="#">Contact</a></li>          
+          </ul>
+      </nav>
+      <div id = "date"></div>
+    </header>
+    <main id = "hidden">
+      <section id = "booklist"></section>
+      </section>
+      <section id = "Add-Books">
+        <div class = "form-center">
+          <form>
+            <h2>Add a new Book</h2>
+            <input type = "text" placeholder="Title" id = "title" >
+            <input type = "text" placeholder="Author" id = "author">
+            <button type="button" id = "add-btn">Add</button>
+          </form>
+        </div>
+      </section>
+      <section id = "contact">
+        <p>Do have any questions or you just want to say "Hello"?
+          You can reach out to us!</p>
+        <ul>
+          <li>Our e-mail: mail@mail.com</li>
+          <li>Our phone number: 00*********</li>
+          <li>Our address: Streetname #,City, Pakistan, Peru & Venezuela.</li>
+        </ul>
+      </section>
+    </main>
```

Comment:
Your design (on page load)
![image](https://github.com/sinruidotec/Awesome-Books.github.io/assets/78251927/93a60be8-aa58-4175-af00-c1f068031413)

Please kindly use this wireframe as a guide in displaying the page content (the main area is to display only one of the three content sections at a time) 🤗

![image](https://github.com/sinruidotec/Awesome-Books.github.io/assets/78251927/abad69e1-3f2a-47a5-aacc-175f913c09b9)